draft-ietf-tls-iana-registry-updates.md

```diff
@@ -218,6 +203,32 @@ update/has updated] the TLS ExtensionType Values registry to:
 | session_ticket                  |         Yes |
 | renegotiation_info              |         Yes |
 
+IANA [SHALL update/has added] the following notes:
+
+Note:
+: Experts are to verify that there is in fact a publicly available
+standard.  An Internet Draft that is posted and never published or a
```

Comment:
s/standard/specification/

Otherwise we say that an I-D is a standard, which makes the entire RFC 2026 process something of a sham.

Reply:
+1